lib/model/container_policies.dart

```diff
@@ -0,0 +1,78 @@
+/*
+ * privacyIDEA Authenticator
+ *
+ * Author: Frank Merkel <frank.merkel@netknights.it>
+ *
+ * Copyright (c) 2024 NetKnights GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import '../utils/object_validator.dart';
+
+part 'container_policies.freezed.dart';
+part 'container_policies.g.dart';
+
+@Freezed(toStringOverride: false, addImplicitFinal: true, toJson: true, fromJson: true)
+class ContainerPolicies with _$ContainerPolicies {
+  static const UNREGISTER_ALLOWED = 'client_container_unregister';
+  static const TOKENS_DELETABLE = 'client_token_deletable';
+  static const ROLLOVER_ALLOWED = 'container_client_rollover';
+  static const INITIAL_TOKEN_TRANSFER = 'container_initial_token_transfer';
+
+  const ContainerPolicies._();
+
+  const factory ContainerPolicies({
+    required bool rolloverAllowed,
+    required bool initialTokenTransfer,
```

Comment:
assignment is probably a better word than transfer because the operation that happens in the server is an assignment